### DISCUSSION TITLE
FMCU import winds from Simbrief

### DISCUSSION THREAD

**Discussion by icywings11** [flybywiresim/aircraft#5019] [Ideas]
The FMCU is great in how it imports the basic Simbrief data. 

Is it hard to also import the climb, cruise, and decent winds? I know we can manually put them in but since all that other data is transferred, how hard would it be to also import that data? 

Just an idea! Keep up the AWESOME work!!!

**Comment by tracernz** [comment]
This will become more of a priority when the guidance system can use winds for lateral path prediction, and eventually VNAV. Stay tuned. 👍
